### ISSUE TITLE
[Bug] No more short key to hide GUI

### ISSUE THREAD

**Issue by bartaback** [MeteorDevelopment/meteor-client#1900]
### Describe the bug

In the last version of Meteor #1388 the shortkey to hide the GUI is gone. Can you re-add it ?

### Steps to reproduce

```bash
Check the GUI part, there's no way to set a shortcut to hide the GUI, there's only a checkbox
```

### Meteor Version

#1388

### Minecraft Version

1.18.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by MineGame159** [comment]
What

**Comment by SIMULATAN** [comment]
I suppose he meant that you could bind the GUI to a key because before it was treated as a module and not a system

**Comment by C10udburst** [comment]
Just use F1

**Comment by bartaback** [comment]
yup, you could bind the display of the GUI with a simple key before, now it's not possible ; F1 is possible indeed, but i like to hide some mods without using F1 (like minimaps and other infos) independantly

**Comment by MineGame159** [comment]
Oh you mean HUD

**Comment by TheRedHippo** [comment]
+1 Please add a bind for toggling HUD. Build 1387 has removed this feature :(

**Comment by arlomcwalter** [comment]
i will do, or someone can just add it in a pr cause i wont be free till later tonight
